src/components/MessageList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+
+
+
+const MessageList = ({ message}) => {
```

Comment:
The `Message List` page does not show any random message. Please make sure two apps work together. Moreover, code in all the components are not formatted.
![blank](https://user-images.githubusercontent.com/35504149/204060919-2f6b5f07-add4-4990-b335-5144136b90c1.png)